backend/prismaCalls.ts

```diff
@@ -235,13 +235,31 @@ export async function checkMetricsExist(metadataId: string): Promise<boolean> {
 
 export async function getDownloadCount(packageId: string): Promise<number> {
     const downloadEntries = await prisma.packageHistoryEntry.findMany({
-      where: {
-        metadata: {
-          id: packageId, // Filtering by the package ID
-        },
+        where: {
+            metadata: {
+            id: packageId, // Filtering by the package ID
+            },
         action: Action.DOWNLOAD, // Filtering by the action "DOWNLOAD"
-      },
+        },
     });
-  
+
     return downloadEntries.length; // Return the length of the filtered entries
-  }
\ No newline at end of file
+}
+
+
+export async function resetDatabase() {
+    try {
+        await prisma.package.deleteMany();
```

Comment:
can you make it so that you dynamically get all the models and then delete everything that way? That way you won't have to update this if we ever update our schema or add any models.